src/Graph.ts

```diff
@@ -242,6 +242,25 @@ export class Graph {
     return people ? people.value : null;
   }
 
+  /**
+   * async promise, returns all planner plans associated with the group id
+   *
+   * @param {string} groupId
+   * @returns {Promise<MicrosoftGraph.PlannerPlan[]>}
+   * @memberof Graph
+   */
+  public async getPlansForGroup(groupId: string): Promise<MicrosoftGraph.PlannerPlan[]> {
+    const scopes = 'people.read';
```

Comment:
These scopes are wrong - please use the right scopes per the [docs](https://docs.microsoft.com/en-us/graph/api/plannergroup-list-plans?view=graph-rest-1.0&tabs=http)

Reply:
Thanks, i probably missed that part :)